common/djangoapps/xblock_django/models.py

```diff
@@ -40,3 +42,42 @@ def disabled_block_types(cls):
             return ()
 
         return config.disabled_blocks.split()
+
+
+class XBlockDeprecatedAdvancedComponentConfig(ConfigurationModel):
```

Comment:
@ehteshamkafeel We use different names in different parts of the code base and "component" is another name for XBlocks. I know we discussed this already but after seeing the code, I am leaning towards moving this field to the `XBlockDisableConfig` model. We can call it `disabled_create_blocks`.